terraform/root_integration.tf

```diff
@@ -0,0 +1,102 @@
+module "jenkins_ecs" {
```

Comment:
Do the module names need "integration" in them, to make it clearer they generate the "integration" environment AWS resources only?

Reply:
Yeah, I think so. I've changed them all to jenkins_integration_xxx